XiEditor/EditViewController.swift

```diff
@@ -158,7 +158,15 @@ class EditViewController: NSViewController, EditViewDataSource {
         lastDragPosition = position
         let flags = theEvent.modifierFlags.rawValue >> 16
         let clickCount = theEvent.clickCount
-        document.sendRpcAsync("click", params: [position.line, position.column, flags, clickCount])
+        if theEvent.modifierFlags.contains(NSCommandKeyMask) {
```

Comment:
My feeling is that it's "ok for now," ie there isn't any other gesture I want to map to those other modifier chords. The idea is that this code will evolve into a nice tree of `.contains` predicates bottoming out into gesture identifiers.